src/libcore/intrinsics.rs

```diff
@@ -11,7 +11,7 @@
 //! In order to make an intrinsic usable at compile-time, one needs to copy the implementation
 //! from https://github.com/rust-lang/miri/blob/master/src/shims/intrinsics.rs to
 //! `librustc_mir/interpret/intrinsics.rs` and add a
-//! `#[rustc_const_unstable(feature = "foo", issue = "01234")]` to the intrinsic.
+//! `#[rustc_const_unstable(feature = "foo2", issue = "01234")]` to the intrinsic.
```

Comment:
Hu?

Reply:
This is part of 84f1d73

Reply:
But why does that require renaming `foo` to `foo2`? (Maybe `bar` would be more on-brand.^^)

Reply:
I fixed up the other use of `foo` so they are aligned.